packages/relay/environment.ts

```diff
@@ -10,10 +10,11 @@ import {
   type Variables,
 } from "relay-runtime";
 
-const HTTP_ENDPOINT = "https://gql.dev.kamp.us/graphql";
 const IS_SERVER = typeof window === typeof undefined;
 const CACHE_TTL = 5 * 1000; // 5 seconds, to resolve preloaded results
 
+const HTTP_ENDPOINT = IS_SERVER ? (process.env.NEXT_PUBLIC_GQL_URL as string) : "/gql";
```

Comment:
This line requires apps using this package to have a rewrite rule configured for `/gql` url to real graphql url (eg `https://gql.dev.kamp.us/graphql`)